integration_tests/features/environment.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+
+def before_all(context):
+    os.system("mkdir mock/temp")
```

Comment:
This makes the `after_all` with `rm` not necessary, since it will be different directories.
```
❯ mktemp -d integration_tests/mock/test_XXXXXX
integration_tests/mock/test_BlEPZP
```

Reply:
We'd still have to create temp dir if we're not using `/tmp`. I think it's OK the way it is right now. Unless you have strong feelings about it. 😉